lix/README.md

```diff
@@ -1 +1,68 @@
-TODO
\ No newline at end of file
+# Lix (previously Project Lisa)
+
+Lix is the git compatible data layer for inlang apps.
```

Comment:
not only inlang apps, but also. "for apps." would be enough I think.

Reply:
I wouldn't write for "inlang apps" here as lix is generic/we want to steer interest from developers that lix is useful for them.